parser/HeadingNode.php

```diff
@@ -17,6 +17,10 @@ class HeadingNode extends Node
 
     public function __construct($data, Node $parent)
     {
+        if (empty($data['content'][0]['text'])) {
```

Comment:
What if I have a headline called `===== 0 =====`?

Reply:
Good point, now checking for empty string only.